assets/training/model_management/src/azureml/model/mgmt/processors/preprocess.py

```diff
@@ -24,11 +28,11 @@ def run_preprocess(mlflow_flavor: str, model_path: Path, output_dir: Path, temp_
     :param preprocess_args: additional preprocess args required by mlflow flavor
     :type preprocess_args: Dict
     """
-    print(f"Run preprocess for model with flavor: {mlflow_flavor} at path: {model_path}")
+    logger.info(f"Run preprocess for model with flavor: {mlflow_flavor} at path: {model_path}")
     if mlflow_flavor == ModelFlavor.TRANSFORMERS.value:
         transformers.to_mlflow(model_path, output_dir, temp_dir, preprocess_args)
     elif mlflow_flavor == ModelFlavor.MMLAB_PYFUNC.value:
         pyfunc.to_mlflow(model_path, output_dir, preprocess_args)
     else:
         raise Exception(f"Unsupported model flavor: {mlflow_flavor}.")
-    print("Model prepocessing completed !!!")
+    logger.info("Model prepocessing completed !!!")
```

Comment:
(nit) Remove exclamations.

Reply:
Also, might be worth adding a decorator that logs the time taken for this and logs the start, end time.

Reply:
Yes tracking that already for download and preprocess.

download: https://github.com/Azure/azureml-assets/blob/ayushmishra/model_import_appinsight_integration/assets/training/model_management/src/azureml/model/mgmt/downloader/download_utils.py#L103C14-L103C14 

preprocess: https://github.com/Azure/azureml-assets/blob/ayushmishra/model_import_appinsight_integration/assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convert.py#L13